CONTRIBUTING.md

```diff
@@ -85,7 +85,7 @@ Kedro-Viz uses an unique identifier to determine the data source. You can config
 DATA=random npm start
 ```
 
-There are several different data sources available. By default in production, the app asynchronously loads JSON from the `/api/main` endpoint. You can replicate this in development by placing a dataset in `/public/api/main`, using `main` as the name of the JSON file, without an extension. Alternatively, you can use one of the mock unit-testing/demo datasets, or pseudo-random data procedurally-generated on page load, which is often useful for local development.
+There are several different data sources available. By default in production, the app asynchronously loads JSON from the `/api/main` endpoint. You can replicate this in development by placing a JSON dataset in `/public/api/main`, using `main` as the name of the file, without an extension. Note that operating systems often add hidden file extensions, so you might need to use a CLI to confirm the filename. Alternatively, you can use one of the mock unit-testing/demo datasets, or pseudo-random data procedurally-generated on page load, which is often useful for local development.
```

Comment:
I like this, but am left confused (a common occurrence!).

It says I need to have a filename `main` without extension, and then adds that there may be a hidden file extension. If there is, what do I do? Remove it? How do I use the mock unit-testing dataset? Is this in the docs somewhere, and if so, maybe we could point to it for those wanting more info?

Reply:
> If there is, what do I do? Remove it?

I debated adding more detailed information, but it depends so much on what operating system you're using, and on your level of experience, that it would be too much information to include. So my goal here was to remind users that this is a possibility, not necessarily to teach them how. I could potentially link to another resource with this info though, e.g. https://www.computerhope.com/issues/ch002089.htm

> How do I use the mock unit-testing dataset? Is this in the docs somewhere, and if so, maybe we could point to it for those wanting more info?

Yeah this is in the following/preceding paragraphs! The section as a whole should be enough info to tell someone how to use the different datasets.

Reply:
It feels like there's a lot going on for one paragraph, maybe could this be broken into a few bullet points?

Reply:
@bru5 Okay, I've rewritten the whole section to hopefully break things up a little better